ui/eureka/eureka.js

```diff
@@ -2067,7 +2071,7 @@ class EurekaTracker {
 
         let remainingMinutes = Math.ceil(remainingMs / 1000 / 60);
         let nmString = respawnIcon + remainingMinutes +
-            this.Translate(this.options.timeStrings.minute);
+            this.TransByLocale(this.options.timeStrings.minute);
```

Comment:
Why are these translated by locale vs by parser language? Can you leave this as `Translate` for now, and if you want to follow up in another pull request with adding a "display language" for everything (vs just assuming that somebody wants the system locale), we can add more options (maybe a top level one?), like raidboss has alert/timeline language options.